integration-tests/validation/src/testDebug/kotlin/com/android/designcompose/testapp/validation/Scrolling.kt

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.android.designcompose.testapp.validation
+
+import androidx.compose.ui.geometry.Offset
+import androidx.compose.ui.test.junit4.createComposeRule
+import androidx.compose.ui.test.onNodeWithText
+import androidx.compose.ui.test.performTouchInput
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import com.android.designcompose.TestUtils
+import com.android.designcompose.test.internal.captureRootRoboImage
+import com.android.designcompose.test.internal.designComposeRoborazziRule
+import com.android.designcompose.testapp.common.InterFontTestRule
+import com.android.designcompose.testapp.validation.examples.ScrollingTest
+import com.github.takahirom.roborazzi.RobolectricDeviceQualifiers
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.robolectric.annotation.Config
+import org.robolectric.annotation.GraphicsMode
+
+@RunWith(AndroidJUnit4::class)
+@GraphicsMode(GraphicsMode.Mode.NATIVE)
+@Config(qualifiers = RobolectricDeviceQualifiers.MediumTablet)
+class Scrolling {
+    @get:Rule val clearStateTestRule = TestUtils.ClearStateTestRule()
+    @get:Rule val composeTestRule = createComposeRule()
+    @get:Rule val roborazziRule = designComposeRoborazziRule(javaClass.simpleName)
+    @get:Rule val interFontRule = InterFontTestRule()
+
+    @Test
+    fun scrollTests() {
+        with(composeTestRule) {
```

Comment:
Is squoosh enabled for this test?

Reply:
I turned on squoosh, and fixed the test by using replacement content and putting test tags on the content.